### ISSUE TITLE
Waiting time

### ISSUE THREAD

**Issue by ahmaddxb** [vabene1111/recipes#461]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by DaFlowah** [comment]
I had the same thought a while back, but then I realized "waiting time" could actually be a valid wildcard, as not all recipes have a "cooking time".
For example:
- pizza dough has to rise several hours (depending on the recipes, 24-48 h)
- fresh pasta dough has to rise 30-60 m
- tiramisu has to be kept in the fridge for about 2 h before serving it
Etc (yes, if you are wondering, I'm Italian).

So, all this to say: "waiting time" could actually be a valid choice. Or there should be different fields for "waiting time" and "cooking time".

**Comment by auanasgheps** [comment]
Individual steps can already be counted as "time", but they are not accounted in the label under the recipe name.

I'm thinking "waiting time" could have a field in case user would like to rename it,  similarly with "servings". Should also be easy to implement.

**Comment by quamok** [comment]
I think waiting time is a valid field, as for example letting sit something for a couple of hours, but I think it should be different than cooking time, as some recipes requires both. My opinion is that a 3rd time field should be added regarding cooking time.

**Comment by Dremor** [comment]
What about the recipes that have cooking and waiting time, like a cheesecake ? Having a differentiated field between cooking and waiting would be useful for sure.

In my example, I have to cook for 40 min, then wait at least 4 hours.

**Comment by supaeasy** [comment]
I agree very much. This is not implemented well. There are four options to describe time according to [schema.org](https://schema.org/Recipe):

- cookTime 
 -- The time it takes to actually cook the dish, in ISO 8601 duration format.
- performTime
-- The length of time it takes to perform instructions or a direction (not including time to prepare the supplies), in ISO 8601 duration format.
- prepTime
-- The length of time it takes to prepare the items to be used in instructions or a direction, in ISO 8601 duration format.
- totalTime
-- The total time required to perform instructions or a direction (including time to prepare the supplies), in ISO 8601 duration format.

**Comment by vabene1111** [comment]
thanks for the information. When I originally implemented this I did not know about schema.org's standard for recipes. At some point this will be rewritten.
